### PR TITLE
mobile-workSections

### PR DESCRIPTION
**Pull request by maths29**
in this project I:
update the contact form in index.html
update the style.css and 
I added the formspree link to the form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,320 +1,426 @@
...
+            <label for="message"></label>
+            <textarea
+              id="message"
+              name="message"
+              placeholder="Enter text here"
+              maxlength="500"
+              required
+            ></textarea>
+            <button type="submit" class="submit">Get in touch</button>
+          </form>
```
Comment (Johnadibe) [inline index.html]: > - [ ] Great job to you both👍 Please kindly ensure that the `input` font size stick to the design templates in Figma as much as possible and also the decorator background image close to the "Get in touch" `button`.😊
> 
> Figma design👇:
> 
> contact-figma
> 
> Your design👇:
> 
> contact-team
>

File: index.html [hunk 2]
```diff
@@ -1,320 +1,426 @@
...
+            />
+            <label for="message"></label>
+            <textarea
+              id="message"
+              name="message"
+              placeholder="Enter text here"
+              maxlength="500"
+              required
+            ></textarea>
+            <button type="submit" class="submit">Get in touch</button>
```
Comment (Johnadibe) [inline index.html]: > - [ ] Please kindly implement the hover interaction of the button state by adding its `box-shadow` to match the Figma design👍
> 
> Figma design👇:
> 
> btn-hover
> 
> Your design👇:
> 
> btn
> 
> 
>

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️
### Hi @maths29 and @Fombi-Favour👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Highlights
✅ You used GitHub Flow correctly 👍🏽
✅ Linter checks passed✔️
✅ Used Flexbox to lay out the elements on the page 👍🏽 
✅ Collect data by using the Formspree service✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -1,320 +1,426 @@
...
```
Comment (usorfaitheloho) [inline index.html]: Please I'd suggest your design follows the Figma template as closely as possible, the `placeholder` text is different from the one we have in the design, kindly re-adjust it to the placeholder text present in the design given so as to have a better design 😄

File: style.css [hunk 4]
```diff
@@ -321,9 +376,8 @@ ul ul {
   color: #fff;
   background-image: url(./img/cfp.png);
   background-repeat: no-repeat;
-  background-position: absolute;
-  background-size: 100% 100%;
-  min-width: 375px;
+  background-position: top right;
+  background-size: contain;
```
Comment (usorfaitheloho) [inline style.css]: Please as suggested by the previous reviewer, kindly adjust the image on the background of the form to be in a position similar to what we have in the design given. yours is at the end of the page but what is in the design is close to the `texxtarea` 👍 

figma design :

image

Your design :

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @maths29  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Implemented the hover button interaction ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tugsem** [comment]
## STATUS: APPROVED 👍🎯  
Hello @maths29 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ correct use of github flow
✔️ linters are passing
✔️ professional documentation
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**